### PR TITLE
Primary and Docket header components and refactor CoA

### PR DESCRIPTION
**Pull request by matthopson**
Separate PDF header elements and update styles per PDF Style guide in Figma.

Screen Shot 2020-04-30 at 2 22 01 PM

### CODE REVIEW

File: shared/src/business/utilities/generateHTMLTemplateForPDF/generateChangeOfAddressTemplate.test.js [hunk 1]
```diff
@@ -108,12 +106,13 @@ describe('generateChangeOfAddressTemplate', () => {
       },
     });
 
+    console.log(result);
```
Comment (rachaelparris) [inline shared/src/business/utilities/generateHTMLTemplateForPDF/generateChangeOfAddressTemplate.test.js]: Could you remove this log please?

File: shared/src/business/utilities/pdfGenerator/components/DocketHeader.jsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+const React = require('react');
+
+const DocketHeader = ({ caption, docketNumberWithSuffix, h3 }) => {
+  return (
+    <>
+      <div className="case-information">
+        <div id="caption">
+          <div id="caption-title">{caption}</div>
+          <div id="caption-petitioners">Petitioner(s)</div>
```
Comment (rachaelparris) [inline shared/src/business/utilities/pdfGenerator/components/DocketHeader.jsx]: I don't think `Petitioner(s)` can be hardcoded. This part of the string is different depending on the party type. Also `caption` throughout the system now refers to the full caption with `, Petitioner` appended, so I'd like to keep that naming convention straight and change this variable to `caseTitle`.

**Review by rachaelparris — CHANGES_REQUESTED** [review]

File: shared/src/business/utilities/pdfGenerator/components/DocketHeader.jsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+          <div id="caption-v">v.</div>
+          <div id="caption-commissioner">Commissioner of Internal Revenue</div>
+          <div id="caption-respondent">Respondent</div>
```
Comment (rachaelparris) [inline shared/src/business/utilities/pdfGenerator/components/DocketHeader.jsx:17]: Should there be some commas in here, before Respondent and before Petitioner? Or are we intentionally leaving out the commas?

Comment (matthopson) [reply shared/src/business/utilities/pdfGenerator/components/DocketHeader.jsx:17]: No commas
Screen Shot 2020-04-30 at 5 04 01 PM

File: shared/src/business/utilities/pdfGenerator/components/DocketHeader.jsx (hunk 2 above)
Comment (matthopson) [reply shared/src/business/utilities/pdfGenerator/components/DocketHeader.jsx]: Man what a ride. I'll be doing some further refactors on this, as it ain't very sustainable.

**Review by rachaelparris — APPROVED** [review]

**Pull request merged**